src/lib/Post.svelte

```diff
@@ -23,10 +23,10 @@
 		var userName = ""
 		if (post.user == "Discord") {
 			post.user = post.content.split(":")[0];
+			post.content = post.content.slice(post.content.indexOf(":")+2);
```

Comment:
would this work a bit better?
![image](https://user-images.githubusercontent.com/61485190/188515904-a0c8d475-28c1-4c8b-add7-f9d02c2cc01b.png)

Reply:
actually yeah

Reply:
for post.content? no. because if i did this: hi: hi on discord, it would render as hi

Reply:
wait, it would be `[1]` not `[0]`, whoops